packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -468,43 +470,54 @@ class RadioListTile<T> extends StatelessWidget {
   /// Defaults to false.
   final bool useCupertinoCheckmarkStyle;
 
+  /// Controls the scaling factor applied to the [Radio] within the [RadioListTile].
+  ///
+  /// Defaults to 1.0.
+  final double radioScaleFactor;
+
   @override
   Widget build(BuildContext context) {
     final Widget control;
     switch (_radioType) {
       case _RadioType.material:
         control = ExcludeFocus(
-          child: Radio<T>(
-            value: value,
-            groupValue: groupValue,
-            onChanged: onChanged,
-            toggleable: toggleable,
-            activeColor: activeColor,
-            materialTapTargetSize: materialTapTargetSize ?? MaterialTapTargetSize.shrinkWrap,
-            autofocus: autofocus,
-            fillColor: fillColor,
-            mouseCursor: mouseCursor,
-            hoverColor: hoverColor,
-            overlayColor: overlayColor,
-            splashRadius: splashRadius,
+          child: Transform.scale(
```

Comment:
We should not add a `Transform.scale()` widget when the `scale` is 1.0.

I don't immediately see the implementation of `RenderTransform` short circuit if the transformation matrix is the identity matrix

https://github.com/flutter/flutter/blob/bae79714317430baa8e9d2edeb6accdaedfd5e06/packages/flutter/lib/src/rendering/proxy_box.dart#L2359

Since it pushes a layer, I would rather just avoid it if it is redundant.

This was also not applied in https://github.com/flutter/flutter/commit/bae79714317430baa8e9d2edeb6accdaedfd5e06

but unfortunately, it looks like that slipped past review there.